docs/fundamentals/toc.yml

```diff
@@ -0,0 +1,1323 @@
+        - name: Regular expression options
+          href: ../standard/base-types/regular-expression-options.md
+        - name: Miscellaneous constructs
+          href: ../standard/base-types/miscellaneous-constructs-in-regular-expressions.md
+      - name: Best practices for regular expressions
+        href: ../standard/base-types/best-practices.md
+      - name: Regular expression object model
+        href: ../standard/base-types/the-regular-expression-object-model.md
+      - name: Regular expression behavior
+        items:
+        - name: Overview
+          href: ../standard/base-types/details-of-regular-expression-behavior.md
+        - name: Backtracking
+          href: ../standard/base-types/backtracking-in-regular-expressions.md
+        - name: Compilation and reuse
+          href: ../standard/base-types/compilation-and-reuse-in-regular-expressions.md
+        - name: Thread safety
+          href: ../standard/base-types/thread-safety-in-regular-expressions.md
+      - name: Examples
+        items:
+        - name: Scan for HREFs
+          href: ../standard/base-types/regular-expression-example-scanning-for-hrefs.md
+        - name: Change date formats
+          href: ../standard/base-types/regular-expression-example-changing-date-formats.md
+        - name: Extract a protocal and port number from a URL
+          href: ../standard/base-types/how-to-extract-a-protocol-and-port-number-from-a-url.md
+        - name: Strip invalid characters from a string
+          href: ../standard/base-types/how-to-strip-invalid-characters-from-a-string.md
+        - name: Verify that strings are in valid email format
+          href: ../standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md
+  - name: Parse strings
+    items:
+    - name: Overview
+      href: ../standard/base-types/parsing-strings.md
+    - name: Parse numeric strings
+      href: ../standard/base-types/parsing-numeric.md
+    - name: Parse date and time strings
+      href: ../standard/base-types/parsing-datetime.md
+    - name: Parse other strings
+      href: ../standard/base-types/parsing-other.md
+  - name: Attributes
+    items:
+    - name: Overview
+      href: ../standard/attributes/index.md
+      displayName: attributes
+    - name: Apply attributes
+      href: ../standard/attributes/applying-attributes.md
+    - name: Write custom attributes
+      href: ../standard/attributes/writing-custom-attributes.md
+    - name: Retrieve information stored in attributes
+      href: ../standard/attributes/retrieving-information-stored-in-attributes.md  
+  - name: Choose between anonymous and tuple types
+    href: ../standard/base-types/choosing-between-anonymous-and-tuple.md
+- name: Microsoft Extensions 
+  items:
+  - name: Configuration
+    href: /aspnet/core/fundamentals/configuration/
+  - name: Dependency injection
+    href: /aspnet/core/fundamentals/dependency-injection/
+  - name: Logging
+    href: /aspnet/core/fundamentals/logging/
```

Comment:
These need to be ported over from https://github.com/dotnet/AspNetCore.Docs to our docs - right? I'll create an issue to capture that effort.

Reply:
We have one already - https://github.com/dotnet/docs/issues/4567